### DISCUSSION TITLE
What's the use of uniqueID in Modal

### DISCUSSION THREAD

**Discussion by pranshuchittora** [twilio-labs/paste#2726] [General] [answered]
Hey,
I am trying to use the Modal component, the usage example codes uses `useUID` hook. I am assuming that it generates an unique identifier (UUID)
1. Why do we need this, and what happens if we skip this
2. Form which lib/package this `useUID` has been imported and used.
3. Can we update the docs on the website about the same

**Comment by gloriliale** [comment, marked as answer]
Hi @pranshuchittora!

[This page of our docs site](https://paste.twilio.design/core/libraries/uid-library) explains more about UIDs, and includes examples of them being used through the package that's included in Paste. For Modal's examples, we are using the UIDs to connect form elements and related parts of the DOM (a Modal and the ModalHeading used to label it). Hope this helps!
